docker-compose.yml

```
@@ -10,7 +10,7 @@ services:
       context: .
       # dockerfile is relative to the context
       dockerfile: joystream-node.Dockerfile
-    container_name: joystream-node
+    container_name: joystream-node-giza
```

```suggestion
    container_name: joystream-node
```

There is no need to rename the container name really.